dora/core/server/worker/src/main/java/alluxio/worker/dora/PagedDoraWorker.java

```diff
@@ -736,7 +738,8 @@ private void invalidateFileMeta(String path) {
   }
 
   @Override
-  public void completeFile(String path, CompleteFilePOptions options, String uuid) {
+  public void completeFile(String path, CompleteFilePOptions options, String uuid)
+      throws IOException, AccessControlException {
```

Comment:
how about the rest operations? Add exception next time?

Reply:
maybe next time, I can see there will be more APIs added.